ts/industries.ts

```diff
@@ -195,6 +196,8 @@ export const industryInputLabels: Partial<Record<IndustryName, FourStrings>> = {
     'GoldMine': ['Coal', 'Beams', 'Rails', 'Tool Crates'],
     'StampMill': ['Gold Ore', 'water', 'Coal', 'Cordwood'],
     'GoldSmelter': ['Refined Gold', 'Coal', input3, input4],
+    'coaltower': ['Coal', input2, input3, input4],
+    'firewooddepot': Cordwood,
```

Comment:
```suggestion
    'firewooddepot': ['Beams', 'Cordwood', 'Lumber', 'Logs'],
```

Reply:
will this leave the input as any of those ? as input 1

Reply:
this will end in error of "cordwood defined but not used" if used with 
`const Cordwood: FourStrings = ['Beams', 'Cordwood', 'Lumber', 'Logs'];` or 
`const Cordwood: FourStrings = ['Beams, Cordwood, Lumber, Logs', output2, output3, output4];`